### PR TITLE
Sheet option for 'cat' command

### PR DESCRIPTION
**Pull request by anuveyatsu**
This PR allows users to 'cat' excel files with sheet option (index or name). E.g., you can try:

* `data cat my.xlsx` - prints the first sheet to stdout
* `data cat my.xlsx --sheet 2` - prints the second sheet to stdout (using sheet index)
* `data cat my.xlsx --sheet mysheet` - prints the sheet named 'mysheet' to stdout
* `data cat my.xlsx out.csv --sheet 2` - saves the second sheet to `out.csv` file
  * same for markdown, html and excel files (e.g., you want to save the second sheet in the separate excel file)

### CODE REVIEW

File: bin/data-cat.js [hunk 1]
```diff
@@ -75,8 +75,12 @@ const dumpIt = async (res) => {
 if (pathParts.name === '_' || (!pathParts.name && process.stdin.constructor.name === 'Socket')) {
   dumpIt(process.stdin)
 } else if (pathParts.name) {
+  // Check both 'sheet' and 'sheets' args as users can use both of them:
```
Comment (zelima) [inline bin/data-cat.js:78]: :+1:

**Review by zelima — APPROVED** [review]
Looks fine

**Comment by zelima** [comment]
@anuveyatsu actually, what about updating docs with they can use both `--sheet` and `--sheets`

**Comment by anuveyatsu** [comment]
@zelima we're handling args separately for cat and push commands. We're planning to re-design the system in the future to make it centralised.

**Review by Mikanebu — APPROVED** [review]
Good! Tested all cases

**Pull request merged**